domain_config/src/Tests/DomainConfigHomepageTest.php

```diff
@@ -48,10 +50,22 @@ function testDomainConfigHomepage() {
     ));
     $homepages = $this->getHomepages();
     foreach ($domains as $domain) {
+      // Clone the site configuration to set individual home pages if different
+      // than default
+      if ($homepages[$domain->id()] != 'node') {
```

Comment:
These should be picked up from `/tests/modules/domain_config_test/config/install`. Is that not happening?